src/fu-daemon.c

```diff
@@ -231,6 +231,27 @@ fu_daemon_start(FuDaemon *self, GError **error)
 	g_return_val_if_fail(FU_IS_DAEMON(self), FALSE);
 	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
 
+	/* super useful for debugging */
+	if (g_getenv("FWUPD_VERBOSE") != NULL) {
+		GHashTableIter iter;
+		const gchar *key;
+		const gchar *value;
+		g_autoptr(GHashTable) metadata = NULL;
+		g_autoptr(GString) str = g_string_new("versions:");
+
+		metadata = fu_engine_get_report_metadata(priv->engine, error);
+		if (metadata == NULL)
+			return FALSE;
+		g_hash_table_iter_init(&iter, metadata);
+		while (g_hash_table_iter_next(&iter, (gpointer *)&key, (gpointer *)&value)) {
+			if (g_str_has_prefix(key, "CompileVersion") ||
+			    g_str_has_prefix(key, "RuntimeVersion")) {
+				g_string_append_printf(str, "\n - %s=%s", key, value);
+			}
```

Comment:
why just these?  I'd say just add 'em all.

Reply:
Sure, I suppose we're already suuuuuuper verbose with -vv anyway. Fixed.